test/chisel.py

```diff
@@ -190,6 +200,53 @@ def cleanup():
         thread.join()
     return cleanup
 
+def do_tlsalpn_challenges(client, authzs):
+    port = 5001
+    example_key, example_cert = load_example_cert()
+    certs = {'localhost': (example_key, example_cert)}
+    challs = {a.body.identifier.value: get_chall(a, challenges.TLSALPN01)
+        for a in authzs}
+    chall_certs = {domain: tls_alpn_01_cert(client, c, domain)
+        for domain, c in challs.items()}
+    # TODO: this won't be needed once acme standalone tls-alpn server servers
+    # certs correctly, not only challenge certs.
+    chall_certs['localhost'] = (example_key, example_cert)
+    server = standalone.TLSALPN01Server(("", port), certs, chall_certs)
+    thread = threading.Thread(target=server.serve_forever)
+    thread.start()
+
+    # Loop until the TLSALPN01Server is ready.
+    while True:
+        try:
+            s = socket.socket()
+            s.connect(("localhost", port))
+            client_ssl = SSL.Connection(SSL.Context(SSL.TLSv1_METHOD), s)
+            client_ssl.set_connect_state()
+            client_ssl.set_tlsext_host_name("localhost")
+            client_ssl.set_alpn_protos([b'acme-tls/1'])
+            client_ssl.do_handshake()
+            break
+        except (socket.error, SSL.Error):
+            time.sleep(0.1)
+        finally:
+            s.close()
+
+    for chall_body in challs.values():
+        client.answer_challenge(chall_body, chall_body.response(client.key))
+
+    def cleanup():
+        server.shutdown()
+        server.server_close()
+        thread.join()
+    return cleanup
+
+def load_example_cert():
```

Comment:
Can you provide instructions for how you generated the `test/test-example.key` and `test/test-example.pem` files in case they need to be updated/changed in the future?